docs/end-to-end-local-email-dev.md

```diff
@@ -115,45 +130,56 @@ proper Topic ARN, so you need to do these steps in this order:
 4. In [SES Email Receiving][ses-email-receiving], ensure the rule
    "ses-all-inbound-to-sns" is Active.
 
-
 ### Configure your app to accept emails addressed to your domain
+
 Django and our Relay code have checks to make sure the HTTPS POSTs are for the
 right domain. So, you'll need to set some environment variable values:
 
-* `MOZMAIL_DOMAIN=yourdomain.com`
-* `RELAY_FROM_ADDRESS=relay@yourdomain.com`
+- `MOZMAIL_DOMAIN=yourdomain.com`
+- `RELAY_FROM_ADDRESS=relay@yourdomain.com`
 
-Note again: These are NOT your ngrok.io domain.
+Note again: These are NOT your ngrok.io domain. Any user that wishes to reply will need to have their email verified in the free tier of SES (see ####verify-emails above)
+
+One challenge with making both of these domains the same is that the reply logic assumes there are two different domains. In order to test replies in a local environment, you have two options:
+
+1. Set up an additional domain for the `replies` email address that is configured for SES reply handling (follow same steps as above)
+2. Alter the following line in `emails/views.py`:
+
+   `domain_numerical = get_domain_numerical(domain)` can be replaced with:
+
+   `domain_numerical = get_domain_numerical(domain) if local_address != 'example_mask_address' else 2`.
+
+   This will allow replies that are sent to `replies@yourdomain.com` to be properly forwarded to the user that owns the mask.
 
 ### Set up your AWS SES to send emails FROM your app
+
 The last part of Relay is sending emails FROM the Relay app to the real email
 addresses of the owners of Relay aliases. You will need to create an AWS SES
 Configuration set for your local Relay server. And, while in SES "sandbox"
 mode, you need to add one of your own email addresses as a verified identity.
 
 1. [Create an SES configuration set][create-ses-config].
-   * (All defaults are fine)
+   - (All defaults are fine)
 2. Set the AWS env vars:
-   * `AWS_SES_CONFIGSET`
-   * `AWS_REGION`
-   * `AWS_ACCESS_KEY_ID` *Must be set in the environment, not just in .env*
-   * `AWS_SECRET_ACCESS_KEY` *Also must be set in the environment*
-2. [Create a new verified identity][create-new-identity] email address.
-   * AWS will send you a confirmation link to the address.
-3. Register a local Relay user with this email address.
-4. Create an alias with this Relay user.
+   - `AWS_SES_CONFIGSET`
+   - `AWS_REGION`
+   - `AWS_ACCESS_KEY_ID` _Must be set in the environment, not just in .env_
```

Comment:
suggestion (blocking): Since markdown supports asterisks underscores interchangeably for emphasis, let's leave the characters un-changed as much as we can to reduce code churn in our PRs.